src/legacy/core_plugins/embeddable_api/public/np_ready/public/plugin.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {
+  PluginInitializerContext,
+  CoreSetup,
+  CoreStart,
+  Plugin,
+} from '../../../../../../core/public';
+import { TriggerRegistry, ActionRegistry, EmbeddableFactoryRegistry } from './types';
+import { createEmbeddables, Embeddables } from './api';
+
+export class EmbeddablePublicPlugin implements Plugin<any, any> {
```

Comment:
nit/question: Why `Public`?   I think I'd get this naming if there was a Private counterpart, but I dont' think there is... unless I'm missing it.

Reply:
`Public` because it is in the `*/public` folder, Embeddables does not have a server part, but there could be also `EmbeddableServerPlugin`, if it had one in `*/server` folder.

This is a convention I started in `data` plugin, and I believe EPAM picked it up, too. For consistency, I'm using it here too, although Embeddables does not have a server part.

Reply:
Cool, thanks for the info.